src/middleware.ts

```diff
@@ -54,6 +54,7 @@ export default authMiddleware({
   },
 
   publicRoutes,
+  debug: true,
```

Comment:
```suggestion
  debug: true,
```
Lança um `process.env.PC_DO_ERICKSSON ? true : false` ou verifica se está em ambiente de dev `process.env.NODE_ENV === "development" ? true : false`